tests/cli/test_family.py

```diff
@@ -131,3 +131,33 @@ def test_family_show_raw(clear_db, run_cli_command, get_pseudo_family):
     for option in ['-r', '--raw']:
         result = run_cli_command(cmd_family_show, [option, family.label])
         assert len(result.output_lines) == len(family.nodes)
+
+
+def test_family_show_unit(clear_db, run_cli_command, get_pseudo_family):
+    """Test the `-u/--unit` option."""
+    elements = ['Ar', 'Kr']
+    cutoff_dict = {'normal': {'Ar': {'cutoff_wfc': 50, 'cutoff_rho': 200}, 'Kr': {'cutoff_wfc': 25, 'cutoff_rho': 100}}}
+
+    family = get_pseudo_family(cls=CutoffsPseudoPotentialFamily, elements=elements, cutoffs_dict=cutoff_dict, unit='Ry')
+
+    # Test the default unit (Ry)
+    result = run_cli_command(cmd_family_show, [family.label, '--raw'])
+
+    for index, element in enumerate(elements):
+        cutoffs = family.get_recommended_cutoffs(elements=element)
+        fields = result.output_lines[index].split()
+        assert_almost_equal(cutoffs[0], float(fields[3]))
+        assert_almost_equal(cutoffs[1], float(fields[4]))
+
+    # Test both option strings and several units
+    for option in ['-u', '--unit']:
+        for unit in ['eV', 'hartree', 'aJ']:
+            result = run_cli_command(cmd_family_show, [family.label, option, unit])
+            cutoffs = family.get_recommended_cutoffs(elements='Ar', unit=unit)
+            header_fields = result.output_lines[0].split()
+            assert header_fields[4] == f'({unit})'
+            assert header_fields[4] == f'({unit})'
+
+            values_fields = result.output_lines[2].split()
+            assert round(cutoffs[0], 1) == float(values_fields[3])
+            assert round(cutoffs[1], 1) == float(values_fields[4])
```

Comment:
Would be good to parametrize this test over the various units. Looks like we have more unit tests ;)
Noticed afterwards that before the loop over units you also do a check for the default and there you check all lines for all elements, and for the unit specific test you only check the `Ar` cutoffs. Maybe we can split these tests in two

```suggestion
@pytest.mark.usefixtures(('unit',), (None, 'eV', 'hartree', 'aJ'))
def test_family_show_unit(clear_db, run_cli_command, get_pseudo_family, unit):
    """Test the `-u/--unit` option."""
    elements = ['Ar', 'Kr']
    cutoff_dict = {'normal': {'Ar': {'cutoff_wfc': 50, 'cutoff_rho': 200}, 'Kr': {'cutoff_wfc': 25, 'cutoff_rho': 100}}}

    family = get_pseudo_family(cls=CutoffsPseudoPotentialFamily, elements=elements, cutoffs_dict=cutoff_dict, unit='Ry')

    # Test the default unit (Ry)
    result = run_cli_command(cmd_family_show, [family.label, '--raw'])

    for index, element in enumerate(elements):
        cutoffs = family.get_recommended_cutoffs(elements=element)
        fields = result.output_lines[index].split()
        assert_almost_equal(cutoffs[0], float(fields[3]))
        assert_almost_equal(cutoffs[1], float(fields[4]))

    # Test both option strings and several units
    for option in ['-u', '--unit']:
        options = [family.label, '--raw']
        
        if unit is not None:
            options.extend([option, unit])
            
        result = run_cli_command(cmd_family_show, options)
        cutoffs = family.get_recommended_cutoffs(elements='Ar', unit=unit)
        header_fields = result.output_lines[0].split()
        assert header_fields[4] == f'({unit})'
        assert header_fields[4] == f'({unit})'

        values_fields = result.output_lines[2].split()
        assert round(cutoffs[0], 1) == float(values_fields[3])
        assert round(cutoffs[1], 1) == float(values_fields[4])
```

Reply:
Oh, I only after executing your suggestion saw that you actually had made a GitHub suggestion. Oops! 😅